assets/src/scss/blocks/Accordion/AccordionStyle.scss

```diff
@@ -49,7 +49,7 @@
       font-size: $font-size-sm;
       border-radius: 4px;
       font-weight: var(--font-weight-regular);
-      line-height: 1;
+      line-height: 20px;
```

Comment:
Since this PR requires UAT.. Have you checked with the Design Team if this value fit with the ones [from here](https://www.figma.com/design/9NtbY8n3at8uOEJTsLrETb/P4-Design-System?node-id=62-5979&t=XvOyHY0PoTTg0mdi-4)?

Reply:
Also if we can use any of [these tokens](https://github.com/greenpeace/planet4-master-theme/blob/main/assets/src/scss/base/_tokens.scss#L6-L37)?

Reply:
> Since this PR requires UAT.. Have you checked with the Design Team if this value fit with the ones [from here](https://www.figma.com/design/9NtbY8n3at8uOEJTsLrETb/P4-Design-System?node-id=62-5979&t=XvOyHY0PoTTg0mdi-4)?

The value (20px) matches with the mockup:

![P4-Design-System-–-Figma](https://github.com/user-attachments/assets/aaf7c862-707c-46d4-9e83-0f956fca7dd9)